src/RestApi/StoreApi/Controllers/Cart.php

```diff
@@ -103,8 +103,57 @@ public function register_routes() {
 				'schema' => [ $this, 'get_public_item_schema' ],
 			]
 		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/select-shipping',
+			[
+				'args'   => [
+					'shipping_rates' => [
+						'description' => __( 'Array of selected shipping rates ids.', 'woo-gutenberg-products-block' ),
+						'type'        => 'array',
```

Comment:
So an additional thing you can do for arrays is define `items` - this is where you can define the type of each array item and have it validated. If this is an array of strings you could have:

```
'items' => [
  'type' => 'string'
]
```